iothub/device/src/Message.cs

```diff
@@ -10,14 +10,20 @@
 
 namespace Microsoft.Azure.Devices.Client
 {
+    internal enum StreamDisposalOwnership
+    {
+        User,
```

Comment:
I was thinking `App` and `Sdk`, maybe. You think of it as the user, but the actor is really the device application.

Reply:
That makes sense, I'll rename this to `StreamDisposalResponsibility` with values `App` and `Sdk`.